src/Parser/Shaman/Elemental/Constants.js

```diff
@@ -16,4 +16,9 @@ export const ECHOES = {
   TRESHOLD_PERCENTAGE : 0.3,
 };
 
+export const EXPOSED_ELEMENTS  = {
```

Comment:
If these constants aren't used in other modules you can just define them in the ExposedElements module